visualization/app/codeCharta/ui/customConfigs/customConfigs.component.scss

```diff
@@ -9,6 +9,7 @@
 		font-size: 9pt;
 		text-transform: none;
 		background-color: rgba(230, 230, 230, 0.96);
+		left: 13px;
```

Comment:
Don't use pixel-values to center something

Reply:
https://www.w3schools.com/css/css_align.asp

see Center Vertically - Using Flexbox

Reply:
Presention-mode Button problem is solved.